src/main.rs

```diff
@@ -154,6 +168,13 @@ impl App {
         }
     }
 
+    fn programming_language_from_args(&self, args: &ArgMatches) -> Result<ProgrammingLanguage> {
+        match args.get_one::<String>("PROGRAMMING_LANGUAGE") {
+            Some(s) => Ok(ProgrammingLanguage::from(s.to_owned())),
+            None => Err(anyhow!("No clash handle given")),
+        }
+    }
```

Comment:
Unfortunate error message.

Reply:
Woops, remember it's never the first copy pasta that get you, it's always the second one when the first doesn't work.